misk-hibernate-testing/src/main/kotlin/misk/hibernate/HibernateTestingModule.kt

```diff
@@ -14,7 +14,7 @@ import misk.jdbc.MySqlScaleSafetyChecks
 import misk.jdbc.TruncateTablesService
 import misk.jdbc.VitessScaleSafetyChecks
 import misk.time.ForceUtcTimeZoneService
-import misk.vitess.StartDatabaseService
+import misk.databases.StartDatabaseService
```

Comment:
I'm not a big fan of plural in package name. Can we call it just `database`?

Reply:
Done